vars/provisionNodes.groovy

```diff
@@ -41,12 +42,18 @@ def call(Map config = [:]) {
         new_list.add(node_list[ii])
       }
       nodeString = new_list.join(',')
+      node_cnt = config['node_count']
     } else if (config['node_count'] > node_cnt) {
       echo "${config['node_count']} clients requested."
       error "Only ${node_cnt} clients available!"
     }
   }
 
+  if (env.NO_CI_PROVISIONING != null) {
```

Comment:
When is this different than `NO_CI_TESTING`?  I.e. do we need two different knobs here?

Reply:
It was a knob to inhibit CI the CI provisioning step.

I was thinking that in the future, we will want to have some type of provisioning tests, but still may not run the unit tests because of resource limitations.